recipes/opencv/2.x/conanfile.py

```diff
@@ -189,23 +189,29 @@ def get_lib_name(module):
 
         def add_components(components):
             # TODO: OpenCV doesn't use cmake target namespace
-            for component, requires in components.items():
-                self.cpp_info.components[component].names["cmake_find_package"] = "opencv_" + component
-                self.cpp_info.components[component].names["cmake_find_package_multi"] = "opencv_" + component
-                self.cpp_info.components[component].libs = [get_lib_name(component)]
-                self.cpp_info.components[component].requires = requires
+            for component in components:
+                conan_component = component["target"]
+                cmake_target = component["target"]
```

Comment:
Same name, any other build tool to be considered? (e.g. pkgconfig)

Reply:
Yes same name for conan component as cmake target, to avoid clashes in pkgconfig generator.